src/model.c

```diff
@@ -205,6 +212,9 @@ void RAI_ModelFree(RAI_Model* model, RAI_Error* err) {
   else if (model->backend == RAI_BACKEND_TORCH) {
     RAI_ModelFreeTorch(model, err);
   }
+  else if (model->backend == RAI_BACKEND_ONNXRUNTIME) {
+    RAI_ModelFreeORT(model, err);
+  }
   else {
     RAI_SetError(err, RAI_EUNSUPPORTEDBACKEND, "Unsupported backend\n");
     assert(0);
```

Comment:
better to abort() as assert() does nothing if NDEBUG is defined.

Reply:
> Also, need to check pointer arguments for null values.

I've added a check for `session` that was missing. Can you maybe suggest other spots?